### PR TITLE
Set HTML properties from configuration

### PR DESCRIPTION
**Pull request by gund**
[RFC Text](https://github.com/orchestratora/rfcs/blob/d6b0cf2fddda00b07811e6dfd4e8cc651891fb7d/accepted/0007-html-config-props.md)

https://github.com/orchestratora/rfcs/blob/d6b0cf2fddda00b07811e6dfd4e8cc651891fb7d/accepted/0007-html-config-props.md?plain=1#L1-L54

### CODE REVIEW

File: accepted/0000-html-config-props.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+which is limited to strings only.
+
+It will be useful to allow to set properties of the HTML element which will expand the values
+allowed to be set on the element to any JS objects.
+
+## Detailed Explanation
+
+Add new property called `properties` to the
+[HtmlTagConfig](https://github.com/orchestratora/orchestrator/blob/8eb225cafa10ae71c5d5c0c00cfa0376459f90d9/libs/html-tag/src/lib/html-tag/html-tag-config.ts#L6)
+wich will set given props on the rendered HTML element.
```
Comment (RyuuGan) [inline accepted/0000-html-config-props.md]: Typo: `wich`

File: accepted/0000-html-config-props.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Rationale and Alternatives
+
+It's possible to set props manually by getting rendered HTML element from the `orc-html-tag`
+component API via `HtmlTagComponent.getElement()` but requires imperative code style and
+usage of some callback hook which would be called when the component is rendered which does
+not exist in the Orchestrator API at the moment.
+
+We could also add some sort of hooks API to Orchestrator that will allow to run arbitraty
+JS code during lifecycle of component to enable imerative use-cases
```
Comment (RyuuGan) [inline accepted/0000-html-config-props.md]: Typo: `imerative`

File: accepted/0000-html-config-props.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+(ex. `created`, `destoyed`, etc.).
```
Comment (RyuuGan) [inline accepted/0000-html-config-props.md]: Minor: should we replace `ex.` with `e.g.` ?

**Review by RyuuGan — CHANGES_REQUESTED** [review]

File: accepted/0000-html-config-props.md (hunk 1 above)
Comment (gund) [reply accepted/0000-html-config-props.md]:
```suggestion
which will set given props on the rendered HTML element.
```

File: accepted/0000-html-config-props.md (hunk 2 above)
Comment (gund) [reply accepted/0000-html-config-props.md]:
```suggestion
JS code during lifecycle of component to enable imperative use-cases
```

File: accepted/0000-html-config-props.md (hunk 3 above)
Comment (gund) [reply accepted/0000-html-config-props.md]:
```suggestion
(e.g. `created`, `destoyed`, etc.).
```

**Review by RyuuGan — APPROVED** [review]
LGTM

**Pull request merged**